libxfixes will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxfixes it will fail in the following way:

```
2024/11/12 23:12:43 INFO running step "test/pkgconf"
2024/11/12 23:12:44 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:44 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:44 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:44 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:44 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + basename /home/build/melange-out/libxfixes-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + dev_pkg=libxfixes-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + apk info -L libxfixes-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + grep -q ^Name: usr/lib/pkgconfig/xfixes.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + basename usr/lib/pkgconfig/xfixes.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + lib_name=xfixes uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + echo usr/lib/pkgconfig/xfixes.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + grep -q '^usr/lib/pkgconfig/xfixes.pc$\|^usr/share/pkgconfig/xfixes.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + pkgconf --exists xfixes uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN + pkgconf --print-errors --exists xfixes uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN Package xproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN Perhaps you should add the directory containing `xproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN Package 'xproto', required by 'xfixes', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 WARN Package 'fixesproto', required by 'xfixes', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:44 INFO pod 678279e071f620d267f6f2ddbc16ac0942f0e04dc91ed40d107452498aeb4cf9 terminated
2024/11/12 23:12:44 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:44 INFO   workspace dir: /home/user/tmp/melange-workspace-306578952
2024/11/12 23:12:44 INFO   guest dir: /home/user/tmp/melange-guest-1316629273
2024/11/12 23:12:44 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxfixes] Error 1
```

--- a/libxfixes.yaml
+++ b/libxfixes.yaml
@@ -39,7 +39,11 @@ subpackages:
     dependencies:
       runtime:
         - libxfixes
+        - xorgproto
     description: libxfixes dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxfixes-doc
     pipeline:

--- a/libxfixes.yaml
+++ b/libxfixes.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxfixes
   version: 6.0.1
-  epoch: 1
+  epoch: 2
   description: X11 miscellaneous 'fixes' extension library
   copyright:
     - license: MIT

